projects/swimlane/ngx-dnd/package.json

```diff
@@ -3,8 +3,8 @@
   "description": "Drag and Drop for Angular2 and beyond!",
   "version": "5.1.0",
   "peerDependencies": {
-    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
-    "@angular/core": "^6.0.0-rc.0 || ^6.0.0",
+    "@angular/common": "^6.0.0-rc.0 || ^6.0.0 || ^7.0.0",
```

Comment:
Would it make sense to make these "<8.0.0"?

Reply:
Although needs to also be >6